gtsfm/visualization/view_scene_olsson.py

```diff
@@ -0,0 +1,81 @@
+"""
+Script to render a GTSFM scene using Open3d.
+
+Results must be stored in Carl Olsson's `data.mat` file format.
+
+Authors: John Lambert
+"""
+import argparse
+import os
+from pathlib import Path
+
+import numpy as np
+
+import gtsfm.visualization.open3d_vis_utils as open3d_vis_utils
+from gtsfm.loader.olsson_loader import OlssonLoader
+
+
+TEST_DATA_ROOT = Path(__file__).resolve().parent.parent.parent / "tests" / "data"
+
+
+def view_scene(args: argparse.Namespace) -> None:
+    """Read Olsson Dataset ground truth from a data.mat file and render the scene to the GUI.
+
+    Args:
+        args: rendering options.
+    """
+    loader = OlssonLoader(
+        args.dataset_root,
+        image_extension=args.image_extension,
+        max_frame_lookahead=args.max_frame_lookahead,
```

Comment:
why is max_frame_lookahead relevant here?

Reply:
You’re right, it’s not relevant, but it’s a required arg to the loader. Now I made it a global dummy constant value